### ISSUE TITLE
Al Dhih, Timeworn Claw Cooldown Override

### ISSUE THREAD

**Issue by Chist-Sergey** [PathOfBuildingCommunity/PathOfBuilding#4670]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

I expect from Al Dhih, Timeworn Claw to override all Warcry Skills' cooldown to 4 seconds at all times, even with Urgent Orders Support Gen.

### What is the actual behaviour/value?

In-game, Urgent Orders Support Gem adds 2 extra seconds to Al Dhih, Timeworn Claw's cooldown override.

### How to reproduce the issue

1. Select Enduring Cry skill gem.
2. Add Urgent Orders support gem.
3. Equip Al Dhih, Timeworn Claw.
4. Go to the "Calcs" tab.
5. Select Enduring Cry as Active Skill.
6. Hower to "Skill Cooldown" in "Skill type-specific Stats".
7. Observe the wrong order of added value before value override.

### Character build code

```shell
https://pastebin.com/MhBXSsEq
```

**Comment by Regisle** [comment]
![image](https://user-images.githubusercontent.com/49933620/182551603-3dbd3051-3c4b-4bc3-8e9d-9b526e42daa3.png)
it is 4?

**Comment by QuickStick123** [comment]
This doesn't seem like intended behaviour as cooldown recovery rate doesn't effect it and the tooltip explicitly states it to be set to 4 seconds. I would consider opening a bug report for poe.
![image](https://user-images.githubusercontent.com/31533893/182557711-7ba10301-5265-4e29-bc8c-9f3739a39a81.png)

**Comment by Chist-Sergey** [comment]
> ![image](https://user-images.githubusercontent.com/49933620/182551603-3dbd3051-3c4b-4bc3-8e9d-9b526e42daa3.png) it is 4? edit: didnt read ingame it adds, thats different to how other cooldown overrides work

That's the point! In-game, when I linked Urgent Orders to my Enduring Cry, it became 6 seconds cooldown instead of 4.

**Comment by Chist-Sergey** [comment]
> This doesn't seem like intended behaviour as cooldown recovery rate doesn't effect it and the tooltip explicitly states it to be set to 4 seconds. I would consider opening a bug report for poe. ![image](https://user-images.githubusercontent.com/31533893/182557711-7ba10301-5265-4e29-bc8c-9f3739a39a81.png)

You are right. But Urgent Orders add +2 seconds flat. I have used the Second Wind Support Gem to test this, and it ignores penalty from the Second Wind.
